plugins/komodor.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: komodor
+spec:
+  version: "v1.0.2"
+  homepage: "https://github.com/amitlin/kubectl-komodor"
+  shortDescription: "Komodor plugin for kubectl"
```

Comment:
This is not at all explanatory. a user typing "kubectl krew install komodor" already implies everything here. 

Can you provide a better short/long explanation? And please don't use redundant words like plugin or kubectl (it's obvious)